blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/ScheduleContext.java

```diff
@@ -152,9 +153,20 @@ public ScheduleContext(long currentTimeMillis, Collection<Integer> managedShards
         registerMBean();
     }
 
+    public ScheduleContext(long currentTimeMillis, Collection<Integer> managedShards, String zookeeperCluster) {
+        this.scheduleTime = currentTimeMillis;
+        this.shardStateManager = new ShardStateManager(managedShards, asMillisecondsSinceEpochTicker());
+        ZKShardLockManager lockManager = new ZKShardLockManager(zookeeperCluster, new HashSet<Integer>(shardStateManager.getManagedShards()));
+        lockManager.init(new TimeValue(5, TimeUnit.SECONDS));
```

Comment:
Can you create a constant for this 5 seconds? Also explaining what this value is.